src/Components/Main/Pages/ProfilePage/ProfileInfo/ProfileInfo.js

```diff
@@ -0,0 +1,28 @@
+import classes from './ProfileInfo.module.scss';
+import userPhoto from '../../../../../assets/images/user.png';
+import { Preloader } from '../../../../Common/Preloader/Preloader';
+import { useMounted } from '../../../../../hooks/useMounted';
+
+export const ProfileInfo = ({profile, isFetchingProfile}) => {
+  const mounted = useMounted();
```

Comment:
Seems like mounted hook should not be in the component
The component just takes props and render them, so it is simple component which can be rendered without issues when provided props are correct
I would suggest to add mounted hook in parent component like:
```
{mounted && <ProfileInfo />}
```

Reply:
This is a very good idea, thanks!
Done!